administrator/components/com_admin/script.php

```diff
@@ -6089,9 +6089,8 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
 			'/libraries/vendor/willdurand/negotiation/tests/Negotiation/Tests/NegotiatorTest.php',
 			'/libraries/vendor/willdurand/negotiation/tests/Negotiation/Tests/TestCase.php',
 			'/libraries/vendor/willdurand/negotiation/tests/bootstrap.php',
-			// 4.0.0 to 4.0.3
+			// 4.0.0 to __DEPLOY_VERSION__
```

Comment:
```suggestion
			// 4.0.2 to 4.0.3
```
Not sure if we ever used __DEPLOY_VERSION__ here, and the first version number should be the latest version before that change. Believe me because I was maintaining that thing in the recent months.

Reply:
On the other hand: No need to change that now, I haver to touch that file anyway before the next release.

Reply:
I checked the build script and it seems to go through all files and replace __DEPLOY_VERSION__ with the actual version number. So I decided to use that.

As for the “from” version you probably have a point. I guess this would be the last thing to change before merging / right after merging? I mean, if this gets merged in 4.1.0 instead of 4.0.3 we probably need to change 4.0.0 with the last 4.0.x release.